NeoCoast/pis2019-chat-widget
# Disable send button & change error handling

# :wrench: Bugfixes, :recycle: Refactor

## Disables send button and throws console errors

#### :link: Trello board reference:

- [Tirar el error de token inválido a consola y no renderizarlo](https://trello.com/c/sVT21Ew0/196-tirar-el-error-de-token-inv%C3%A1lido-a-consola-y-no-renderizarlo)

---

#### :pencil2: Description:

Only the input was disabled when we wanted to disable all input buttons.
Also, we are now sending the invalid token error to the console instead of showing a message


#### How to test

- Finish a conversation and at the time of feedback, check you are unable to either send or input emojis or files using any of the buttons.
- Set up an invalid auth token and the error should show up on the console and the widget should not render
